docs/source/community/index.md

```diff
@@ -15,6 +16,7 @@ a [roadmap](target-roadmaps) outlining our current priorities, and a detailed [c
 
 mission-scope
 roadmaps
+people
```

Comment:
I would even put "people" first, at the top of the TOC. It's also the link that is mentioned first on the Community main page.